### PR TITLE
'Authors' and 'Acknowledgements' section separate

### LINKED ISSUE #66: Suggested change : "Authors" and "Acknowledgement" should be separate sections
@katherinepeterson , i think its better to keep authors and acknowledgement sections as separate , as they kinda serve a bit different purpose , also one unassumed benefit would be that the section name also would not be unnecessary too long which might also avoid bloating UI alignment in future .

**My understanding on purpose of sections** :
Authors -- can be used mention the key authors of the codebase/project
Acknowledgements - also mean crediting other libs , projects , of which help/inspiration was taken to build a project . One such example here :  https://github.com/othneildrew/Best-README-Template

### PR DESCRIPTION
**Pull request by saurabhchaturvedy**
fixes #66

### CODE REVIEW

**Review by aravindvnair99 — APPROVED** [review]

File: data/section-templates.js [hunk 1]
```diff
@@ -212,13 +212,25 @@ Add badges from somewhere like: [shields.io](https://shields.io/)
...
   },
-  {
+   {
     slug: 'authors',
-    name: 'Authors and Acknowledgement',
+    name: 'Authors',
+    markdown: `
+## Authors
+
+- [@katherinepeterson](https://www.github.com/katherinepeterson) .
```
Comment (katherinepeterson) [inline data/section-templates.js:221]: there's an unnecessary `.` at the end of this line

**Review by katherinepeterson — CHANGES_REQUESTED** [review]
Looks good, just one small change on the line I commented on.

File: data/section-templates.js [hunk 2]
```diff
@@ -212,13 +212,25 @@ Add badges from somewhere like: [shields.io](https://shields.io/)
 
   `,
   },
-  {
+   {
```
Comment (waldyrious) [inline data/section-templates.js:215]: Was this space added on purpose?

File: data/section-templates.js [hunk 3]
```diff
@@ -212,13 +212,25 @@ Add badges from somewhere like: [shields.io](https://shields.io/)
...
+    slug: 'acknowledgement',
+    name: 'Acknowledgements',
     markdown: `
-## Authors and Acknowledgement
+## Acknowledgements
 
-- [@katherinepeterson](https://www.github.com/katherinepeterson) for development and design.
+- [Awesome Readme Templates](https://awesomeopensource.com/project/elangosundar/awesome-README-templates)
+ - [Awesome README](https://github.com/matiassingers/awesome-readme)
+ - [How to write a Good readme](https://bulldogjob.com/news/449-how-to-write-a-good-readme-for-your-github-project)
```
Comment (waldyrious) [inline data/section-templates.js:233]: Why are these lines indented by one space, whereas the first item in the list is unindented?

**Pull request closed without merging**